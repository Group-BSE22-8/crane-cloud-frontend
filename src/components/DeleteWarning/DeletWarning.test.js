import React from 'react';
import { shallow } from "enzyme";
import DeletWarning from '.';

describe('Test the delete warning component', () => {
    const AlignedDeletWarningComponent = shallow(<DeletWarning textAlignment="center" />);
    const UnalignedDeletWarningComponent = shallow(<DeletWarning textAlignment={false} />);

    it('checks if the delete warning component matches the snapshot', () => {
        expect(AlignedDeletWarningComponent).toMatchSnapshot();
    });
});