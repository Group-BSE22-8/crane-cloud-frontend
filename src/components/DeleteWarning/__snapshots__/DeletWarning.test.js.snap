// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the delete warning component checks if the delete warning component matches the snapshot 1`] = `ShallowWrapper {}`;
